steam_web_api/apps.py

```diff
@@ -102,14 +103,14 @@ def search_games(self, term, country="US"):
         soup = BeautifulSoup(html, features="html.parser")
         links = soup.find_all("a")
         apps = []
-        for l in links:
-            if l.has_attr("data-ds-appid"):
+        for link in links:

```

Comment:
ahh yes definitely better now that more people are committing to the project

Reply:
It's even an automatic flag by linters, from the top of my head, it's something long the line of 'ambiguous variable name'